app/screens/network/Network.tsx

```diff
@@ -138,7 +138,10 @@ const Network = ({ history }) => {
       <DetailsRow>
         <DetailsTextWrap>
           <DetailsText>Age</DetailsText>
-          <Tooltip width={250} text="tooltip age" />
+          <Tooltip
+            width={250}
+            text="​​Elapsed time since the current Network deployment"
```

Comment:
I think that deployment sounds good for devnets, but for the mainnet/testnet "genesis" is a better term, since it should not change after the genesis :)
```suggestion
            text="​​Elapsed time since the current Network genesis"
```